### PR TITLE
Add inactivity timeout option

### LINKED ISSUE #34: Implement a option to set the maximum time of absence till the counter pause
At the moment, this is hardcoded.

### PR DESCRIPTION
**Pull request by zachwills**
Resolves #34 

Screenshot:
screen shot 2016-10-25 at 11 00 41 am

### CODE REVIEW

**Comment by blackus3r** [comment]
Do you have an idea, how to display the tooltips in a better way? The title tag was a small workaround for my option field, but I think it isn't nice.

**Comment by zachwills** [comment]
If you add something with a class of `description` it will be italicized and look like a description. That is pretty common with option pages like this.

By the way, do you know how to mark the variable `pwl` as a global? Codacy keeps yelling about it and the various methods I've tried don't seem to work.

**Comment by blackus3r** [comment]
Have a look at the line 100 

Maybe you could implement it like that, so the code will not crash, if pwl is not defined. 
`if (typeof pwl == 'undefined')         
{            
       var pwl.inactivityTimeout = 5;
}
`

I think, codacy will then stop yelling about it.

**Comment by blackus3r** [comment]
I think, we shoul think about the solution with the `description` class.

**Comment by zachwills** [comment]
I updated the screenshot with the new `description` descriptions.

File: post-worktime-logger.php [hunk 1]
```diff
@@ -275,9 +275,12 @@ function isControlBoxEnabled()
 	if ($hook=="post.php")
 	{
 		wp_enqueue_script("post-worktime-logger", plugins_url( "resources/js/post-worktime-logger.js", __FILE__ ));
-        wp_localize_script( 'post-worktime-logger', 'pwl',
-            array( 'ajax_url' => admin_url( 'admin-ajax.php' ) )
-        );
+        $pwt_options = get_option("post-worktime-logger-options");
```
Comment (blackus3r) [inline post-worktime-logger.php]: I have already loaded the options, you can use `$pwlOptions`

**Review by blackus3r — CHANGES_REQUESTED** [review]
Looks good to me :)

**Comment by zachwills** [comment]
@blackus3r updated to use existing options.

**Review by blackus3r — APPROVED** [review]

**Comment by blackus3r** [comment]
Nice work 👍 Thank you

**Pull request merged**